src/SayMore/UI/ConvertMediaDlgViewModel.cs

```diff
@@ -88,13 +89,6 @@ public string ConversionOutput
 			get { return _conversionOutput.ToString(); }
 		}
 
-		/// ------------------------------------------------------------------------------------
-		public void SetConversionStateBasedOnPresenceOfFfmpegForSayMore()
-		{
-			ConversionState = (FFmpegDownloadHelper.DoesFFmpegForSayMoreExist ?
```

Comment:
ConvertMediaUIState.FFmpegDownloadNeeded (and the code that uses it) can go away completely.

Reply:
Done.